blockly-core/generators/javascript.js

```diff
@@ -108,8 +108,10 @@ Blockly.JavaScript.INFINITE_LOOP_TRAP = null;
 
 /**
  * Initialise the database of variable names.
+ * @param {Array.<Blockly.Block>} opt_blocks Optional blocks to query for
```

Comment:
Think you could bracket this like: `[opt_blocks]` to [mark it optional](http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values)